Automattic/wp-calypso
# Business Hours Block: add description and ensure proper capitalization

#### Changes proposed in this Pull Request

* As noted by @eduardozulian - the days of the week are not capitalized in Brazilian Portuguese, and this PR ensures that we are always capitalized.

* As noted by @jeherve , the business hours block has no description. This PR adds one.

#### Testing instructions

* Use this Jurassic Ninja link, and try adding a business hours block on the post editor.
* Ensure the block has a description.
* Ensure the day names ( Monday, Tuesday, etc ) are capitalized in the post editor
